source/documents/insert.txt

```diff
@@ -79,6 +79,10 @@ To insert documents into your collection:
                     { "_id" : 8751, "title" : "The Banquet", "author" : "Dante", "copies" : 2 }
                   ]
 
+            .. note::
+
+              If an object ID is not provided by user, Compass will automatically generate an object ID.
```

Comment:
It may be helpful to link the word "ObjectId" to the [corresponding page in the manual](https://docs.mongodb.com/manual/reference/bson-types/#objectid)

To do so, you can do:

```
:manual:`ObjectId </reference/bson-types/#objectid>`
```

I can explain this in more detail via Slack, but you can look at the `extlinks` in the `conf.py` file.

Reply:
[note] I'd only do this for the _first_ occurrence of "ObjectId". One link is enough